terraform/variables.tf

```diff
@@ -0,0 +1,13 @@
+variable "domain" {
+  description = "The domain where Rabble Rouser is deployed."
+}
+
+variable "region" {
+  default = "ap-southeast-2"
+}
+
+variable "redirector_lambda_name" {
+  default = "redirector"
+}
+
+variable "api_gateway_stage_name" {}
```

Comment:
What exactly does this do? Does it need to be variable? I'm an API gateway noob 🙂